Yosemite/YosemiteTests/Mockups/MockupImageSourceWriter.swift

```diff
@@ -0,0 +1,13 @@
+@testable import Yosemite
+
+final class MockupImageSourceWriter: ImageSourceWriter {
```

Comment:
I actually created `MockImageSourceWriter` first, and then noticed that the other major mock classes/structs are named `Mockup*` - for example, `MockupNetwork`, `MockupSiteStore`, etc. Personally I prefer `Mock*` as well for similar reasons you listed. Maybe we could rename them in a different issue so that all the mock classes/structs have consistent naming? cc @pmusolino

Reply:
I totally agree! Feel free to change `MockupImageSourceWriter` to `MockImageSourceWriter` in this PR. I created an issue for all the other cases, just to not forget about it. https://github.com/woocommerce/woocommerce-ios/issues/1787